pkgs/applications/video/clapper/default.nix

```diff
@@ -58,20 +54,19 @@ stdenv.mkDerivation rec {
     gtk4
     libGL
     libadwaita
-    libsoup
+    libsoup_3
```

Comment:
Is this still needed? I recalled gst_all_1 directly linked to it's libsoup_3

Reply:
without this, the build will fail